container/Operator.go

```diff
@@ -0,0 +1,263 @@
+/*******************************************************************************
```

Comment:
rename this file and the test file to be lower-case

Reply:
Done